src/components/Header/PageSettings/ContentOptions/ContentOptions.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Dropdown from '../../../Dropdown';
+import { options } from '../../../../config/headerMockups';
+import PropTypes from 'prop-types';
+
+const ContentOptions = (props) => {
+  return (
+    <div className={'container dropdowns'}>
```

Comment:
classname can be set as `<div className="container dropdowns">` just in double-quotes, because there is no any calculated value inside. so just simple string =)

Reply:
!!!